### ISSUE TITLE
Search Regex Error

### ISSUE THREAD

**Issue by philip47** [johngodley/search-regex#15]
```
Plugin: 2.0.1
WordPress: 5.4.1 (single)
PHP: 7.4.6
Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36
JavaScript: https://www.*****.com/wp-content/plugins/search-regex/search-regex.js
REST API: https://www.*****.com/wp-json/

Error: Missing parameter(s): replacePhrase (rest_missing_callback_param)
Raw: {"code":"rest_missing_callback_param","message":"Missing parameter(s): replacePhrase","data":{"status":400,"params":["replacePhrase"]}}
```

**Comment by johngodley** [comment]
Can you explain what you did to see that message?

**Comment by philip47** [comment]
> Can you explain what you did to see that message?

I'm trying to remove a phrase

**Comment by johngodley** [comment]
This is fixed by #18. It will be released in 2.1

You can try it out in the beta here: https://github.com/johngodley/search-regex/releases/tag/2.1
